src/core/lv2/Lv2Proc.cpp

```diff
@@ -528,12 +533,11 @@ void Lv2Proc::initPluginSpecificFeatures()
 	// worker (if plugin has worker extension)
 	Lv2Manager* mgr = Engine::getLv2Manager();
 	if (lilv_plugin_has_extension_data(m_plugin, mgr->uri(LV2_WORKER__interface).get())) {
-		const auto iface = static_cast<const LV2_Worker_Interface*>(
-			lilv_instance_get_extension_data(m_instance, LV2_WORKER__interface));
 		bool threaded = !Engine::audioEngine()->renderOnly();
-		m_worker.emplace(iface, &m_workLock, threaded);
+		m_worker.emplace(&m_workLock, threaded);
 		m_features[LV2_WORKER__schedule] = m_worker->feature();
-		// Note: m_worker::setHandle will still need to be called later
+		// note: the worker interface can not be instantiated yet - it requires m_instance
+		//       see initPlugin()
```

Comment:
Doesn't really matter much, I just think that a single comment is simpler and more readable.
```suggestion
		// note: the worker interface can not be instantiated yet - it requires m_instance. see initPlugin()
```